jdk/make/closed/autoconf/openj9ext-version-numbers

```diff
@@ -0,0 +1,21 @@
+# ===========================================================================
```

Comment:
I'd still like to see this file named `openj9ext-version-numbers.sh` as it is a shell script.

Reply:
The current naming is consistent with OpenJDK conventions which for autoconf files do not have .sh on the end. I suggest you raise a concern on a OpenJDK mail list

Reply:
Fair enough. Outside of autoconf, shell scripts have the .sh suffix.